src/font_family.rs

```diff
@@ -42,25 +47,43 @@ impl FontFamily {
         stretch: FontStretch,
         style: FontStyle,
     ) -> Font {
+        self.try_get_first_matching_font(weight, stretch, style)
+            .unwrap()
+    }
+
+    pub fn try_get_first_matching_font(
```

Comment:
Since Rust idiomatically doesn't use the word `get_` in method names, I wonder if a way to do this would be to make the fallible ones names like `first_matching_font` and then to deprecate the non-fallible versions. That way we could gradually transition to a more idiomatically-Rust-like interface. What do you think?

Reply:
Sounds good to me! Will update.

Reply:
Ok I've updated the names and `#[deprecated]` the old methods.